docs/5.x/matomo-font.md

```diff
@@ -51,7 +51,9 @@ Follow below steps to add, change or remove an icon.
   * get `woff2_compress` e.g. via the woff2 debian package or similar packages for other distributions or compile it yourself as explained on [https://github.com/google/woff2](https://github.com/google/woff2).
   * convert the .ttf file using `woff2_compress matomo.ttf`
   * there should now be a `matomo.woff2`
-* Update [plugins/Morpheus/stylesheets/base/icons.css](https://github.com/matomo-org/matomo/blob/5.x-dev/plugins/Morpheus/stylesheets/base/icons.css) and add a new CSS class for this font. The style content in this class needs to mention the font, for example `\e609`. You find the correct value for this from the `style.css` file of the extracted zip file. Simply search for the icon name there to find the correct value.
+* Update [plugins/Morpheus/stylesheets/base/icons.css](https://github.com/matomo-org/matomo/blob/5.x-dev/plugins/Morpheus/stylesheets/base/icons.css) 
+  * Add a new CSS class for this icon. The style content in this class needs to mention the font character, for example `\e609`. You can find the correct value for this from the `style.css` file of the extracted zip file. Simply search for the icon name there to find the correct value.
+  * Update cachebuster parameter of the four font URLs at the top of the file, for example `url('plugins/Morpheus/fonts/matomo.woff2?rzeu9j')` could become `url('plugins/Morpheus/fonts/matomo.woff2?gt43xz')`. The cache buster string can be any six digit random alphanumeric string as long as it's different from the previous one. Providing a new cache buster value will invalidate any old versions of the font in browser caches and force the new font to be used.  
```

Comment:
I've seen cases when the URL query param didn't invalidate for everyone all the time, there might be some obscure proxies etc, but it does work most of the time. The safest way would be to rename the file each time to have the cache-buster param as a part of the actual filename, but that would be a bit too much I reckon.